src/logger/logger.js

```diff
@@ -0,0 +1,21 @@
+import { createLogger, format, transports } from 'winston';
+import chalk from 'chalk';
```

Comment:
lets use winston 'format.colorize()' for prettifying logs, it is enough for most cases

Reply:
`chalk` package was removed